ext/hash/tests/hash_update_stream_basic_001.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Ensure hash_update_stream() always returns the same hash when $length = 0
+--CREDITS--
+Symeon Charalabides <symeon@systasis.com> - @phpdublin
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
+    die('SKIP - php version too low.');
+}
```

Comment:
This doesn't make a lot of sense, the tests are provided with PHP and are not going to be backported

Reply:
This is an old pull request. Fixed now.